AdminWebsite/AdminWebsite/ClientApp/src/app/dashboard/dashboard.component.html

```diff
@@ -72,4 +72,15 @@ <h1 class="vhpanel-title">Manage work allocation</h1>
       </div>
     </a>
   </div>
+
+  <div class="govuk-grid-column-one-third" *ngIf="showManageTeam">
+    <a class="vhpanel-link" [routerLink]="['/manage-team']">
+      <div class="vhpanel vhpanel-manage-team" id="manageTeamBtn">
```

Comment:
For this, you're using camelCasing, which isn't consistent with how it's being done in the manage-team.component.html further down, which is using kebab-casing

Reply:
I chose the same convention as what the dashboard component is doing